### PR TITLE
Data for running GLM on Cannonsville and Pepacton

### PR DESCRIPTION
**Pull request by aappling-usgs**
This repo collects, subsets, and pushes data relevant to reservoir GLM modeling up to [ScienceBase](https://www.sciencebase.gov/catalog/item/6006eec9d34e592d867201d0). The meteo files are being provided to this repo by Jordan via email. Other files come from lake-temperature-model-prep or delaware-model-prep repos.

### CODE REVIEW

File: remake.yml [hunk 1]
```diff
@@ -5,45 +5,100 @@ packages:
...
+      site_ids = site_ids_drb)
+  
+  out_data/ltmp_nml_list.rds:
+    command: fetch_filter_nml(
+      out_rds = target_name,
+      in_ind = '../lake-temperature-model-prep/7_config_merge/out/nml_list.rds.ind',
+      in_repo = I('../lake-temperature-model-prep/'),
+      site_ids = site_ids_drb)
+      
+  # Not quite robust pipelining - if the files in in_data change
```
Comment (hcorson-dosch-usgs) [inline remake.yml:75]: What does `in_data` refer to here? Files in the `lake-temperature-model-prep` repo?

Comment (hcorson-dosch-usgs) [reply remake.yml:75]: Ah I see - looks like that folder is generated by the fetch_meteo_files task table.

**Review by hcorson-dosch-usgs — APPROVED** [review]
Looks good. Unfortunately, I can't test the code by rebuilding, b/c of the implicit target dependencies from other repos, but I traced through the code to build each of the targets, and everything looks good and is clearly commented.

File: remake.yml (hunk 1 above)
Comment (aappling-usgs) [reply remake.yml:75]: Worse than that - it's actually a manually created, manually populated folder. (At least, i don't remember / couldn't find a `dir.create('in_data')` call, and I know for sure that I hand-copy the NLDAS meteo files into that folder after receiving them from Jordan via Teams or email.)

**Comment by aappling-usgs** [comment]
Thanks for reviewing, @hcorson-dosch ! Jake, Sam, and I agreed this morning that we'll be going with a policy of nearly always getting a PR review (except for completely trivial PRs) and then having the PR creator merge her/his own once she/he is satisfied that they've addressed the comments. So I'm merging now.

**Pull request merged**